tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -135,6 +135,9 @@ def _fused_batch_norm(
     center=True,
     scale=False,
     epsilon=0.001,
+    renorm=False,
+    RMAX=None,
```

Comment:
I have documented it. According to the paper, it is best initialized by using a scalar tensor which increases w/ the learning step value. However, in the init value I have put in, I have set it to 1.0 or 0.0, which is technically batch normalization. Any suggestions for alternative default tensors ?